Description: Some folder are not created before installing files into them:
 fix that.
Author: Didier Raboud <didier@raboud.com>
Origin: vendor
Forwarded: yes (private mail)
Last-Update: 2010-04-19
Index: usb-modeswitch.git/Makefile
===================================================================
--- usb-modeswitch.git.orig/Makefile	2010-04-19 12:19:53.000000000 +0200
+++ usb-modeswitch.git/Makefile	2010-04-19 12:25:48.000000000 +0200
@@ -23,8 +23,11 @@
 install: all
 	install -d $(SBINDIR)
 	install --mode=755 usb_modeswitch $(SBINDIR)/usb_modeswitch
+	install -d $(UDEVDIR)
 	install --mode=755 usb_modeswitch.tcl $(UDEVDIR)/usb_modeswitch
+	install -d $(ETCDIR)
 	install --mode=644 usb_modeswitch.conf $(ETCDIR)/usb_modeswitch.conf
+	install -d $(MANDIR)
 	install --mode=644 usb_modeswitch.1 $(MANDIR)/usb_modeswitch.1
 
 
